Dockerfile

```diff
@@ -1,7 +1,6 @@
 ARG GDAL=ubuntu-small-3.3.3
 ARG PYTHON_VERSION=3.8
 FROM osgeo/gdal:${GDAL} AS gdal
-ARG PYTHON_VERSION
```

Comment:
Keep this here:
```
ARG PYTHON_VERSION=3.8
```

Reply:
Done! Out of curiosity, any particular reason to move that line down?

Reply:
Thanks 👍

Try running this command with the different combinations and see what happens with the defaults:
```
docker build -t rasterio .
```